examples/hosts/app-integration/external-data/src/view/debugView.tsx

```diff
@@ -39,11 +38,33 @@ export const DebugView: React.FC<IDebugViewProps> = (props: IDebugViewProps) =>
     );
 };
 
+function isDeepEqual(obj1: object, obj2: object): boolean {
```

Comment:
Might be worth pulling in an existing deep equals solution here, rather than writing our own. Not sure if we have something like this in our common utilities or anything. If not, there is always [lodash.isEqual](https://www.npmjs.com/package/lodash.isequal).

Reply:
I tried installing that but ran into installation errors. I'll keep trying.

Reply:
Let me know if I can help :)